eagle-core/eagle-data-process/eagle-stream-process-api/src/test/java/org/apache/eagle/datastream/TestExecutionEnvironmentJava.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.eagle.datastream;
```

Comment:
Only problem is that we currently still have some files like sample input data and font files can't pass the standard easily. :-)